src/ontology/mondo-edit.obo

```diff
@@ -586090,6 +586090,16 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-18T18:06:40Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7858" xsd:anyURI
 
+[Term]
+id: MONDO:1040014
+name: PPFIA3-related neurodevelopmental disorder
+def: "A neurodevelopmental disorder caused by variation in the PPFIA3 gene. This disorder is characterised by developmental delay and intellectual disability. Most patients present variable additional features, including dysmorphisms, microcephaly or macrocephaly, hypotonia, autism spectrum disorder or autistic features, abnormal electroencephalogram, and epilepsy." [https://orcid.org/0000-0002-0587-4693, https://orcid.org/0000-0002-0987-9862, PMID:38181735]
+is_a: MONDO:0100500 ! Mendelian neurodevelopmental disorder
+intersection_of: MONDO:0100500 {source="PMID:38181735", source="https://orcid.org/0000-0002-0587-4693", source="https://orcid.org/0000-0002-0987-9862"} ! Mendelian neurodevelopmental disorder
```

Comment:
we currently do not add source annotation for the logical definition. I will use these sources for the SubClassOF annotation